vnext/Microsoft.ReactNative/IJSValueReader.idl

```diff
@@ -3,10 +3,11 @@
 
 #include "DocString.h"
 
-namespace Microsoft.ReactNative {
-
-  // Type of value read from JavaScript by IJSValueReader
-  enum JSValueType {
+namespace Microsoft.ReactNative
+{
+  DOC_STRING("Type of value read by IJSValueReader.")
```

Comment:
moving the comments from the markdown PR to here :)
```suggestion
  DOC_STRING("Type of value read by @IJSValueReader.")
```

Reply:
Updated. Thank you!